### ISSUE TITLE
 Only inherit themes which are really needed

### ISSUE THREAD

**Issue by alexxcons** [zayronxio/Zafiro-icons#134]
Hi Zafiro team,

now that the [related xdg merge request](https://gitlab.freedesktop.org/xdg/default-icon-theme/-/merge_requests/5) got merged and v0.18 of the spec got released, I am starting my mission to get the "Inherits" attribute right for popular icon-themes to prevent potentially missing icons 🙂

[Currently Zafiro Light has](https://github.com/zayronxio/Zafiro-icons/blob/master/Light/index.theme#L6):

> Inherits=Surfn,Adwaita,Numix,Numix-Circle-Light,Numix-Circle,breeze,gnome,hicolor

[Zafiro Dark has](https://github.com/zayronxio/Zafiro-icons/blob/master/Dark/index.theme#L6):

> Inherits=hicolor,Adwaita,Surfn,Numix,Numix-Circle,breeze-dark,gnome-dark

I wonder why it inherits from so many other themes ... is Zafiro lacking fd.org icons which need to be provided by one of these?

If there is no strong reason to keep them in the Inherits list, I would suggest dropping all but `hicolor`.

If one/some of the other themes need to stay on the list for some reason, it makes sense to have a section in the README.md to explain packagers that the `zafiro-icon-theme` package should have a hard dependency on the other listed themes.

**Comment by vyvir** [comment]
I agree, but I would personally also suggest to use `Inherits=Adwaita,hicolor` instead, as hicolor tends to make a lot of icons look ugly and Adwaita prevents that.
